nrf_security/Kconfig

```diff
@@ -173,18 +173,14 @@ menuconfig NRF_SECURITY_RNG
 if NRF_SECURITY_RNG
 
 config MBEDTLS_CTR_DRBG_C
-	bool
-	prompt "PRNG using CTR_DRBG"
-	default y
+	bool "PRNG using CTR_DRBG"
```

Comment:
Nothing wrong on this line, but comments cannot be place on the commit message:
https://github.com/nrfconnect/sdk-nrfxlib/pull/288/commits/ec31b1998947cabd9691b27e87139711c035ebc6

Please spellcheck the commit message title:
```
nrf_security: Optimize momory footprint
```

Reply:
It looks like only OpenThread samples uses nrf_security and as you's seen the changes have been addressed. There is also SPM_SECURE_SERVICES but it uses NRF_SECURITY_RNG which hasn't been changed.
For customers the release notes should be enough and they will get lower memory footprint in return.

Reply:
I can obviously disable unnecessary nrf_security options only for OpenThread samples, but this would be a walk around fix which I believe we don't aim into.